code/datums/storage/subtypes/pockets.dm

```diff
@@ -108,61 +119,80 @@
 		/obj/item/toy/crayon/spraycan)
 		)
 
-/datum/storage/pockets/shoes/clown/New()
+/datum/storage/pockets/shoes/clown/New(
+	atom/parent,
+	max_slots,
+	max_specific_storage,
+	max_total_storage,
+)
 	. = ..()
-	set_holdable(list(
-		/obj/item/knife,
-		/obj/item/spess_knife,
-		/obj/item/switchblade,
-		/obj/item/pen,
-		/obj/item/scalpel,
-		/obj/item/reagent_containers/syringe,
-		/obj/item/dnainjector,
-		/obj/item/reagent_containers/hypospray/medipen,
-		/obj/item/reagent_containers/dropper,
-		/obj/item/implanter,
-		/obj/item/screwdriver,
-		/obj/item/weldingtool/mini,
-		/obj/item/firing_pin,
-		/obj/item/suppressor,
-		/obj/item/ammo_box/magazine/m9mm,
-		/obj/item/ammo_box/magazine/m10mm,
-		/obj/item/ammo_box/magazine/m45,
-		/obj/item/ammo_casing,
-		/obj/item/lipstick,
-		/obj/item/clothing/mask/cigarette,
-		/obj/item/lighter,
-		/obj/item/match,
-		/obj/item/holochip,
-		/obj/item/toy/crayon,
-		/obj/item/bikehorn,
-		/obj/item/reagent_containers/cup/glass/flask),
-		list(/obj/item/screwdriver/power,
-		/obj/item/ammo_casing/rocket,
-		/obj/item/clothing/mask/cigarette/pipe,
-		/obj/item/toy/crayon/spraycan)
-		)
+	set_holdable(
+		can_hold_list = list(
+			/obj/item/knife,
```

Comment:
would be nice to alphabetize this while we're fixing the multilining (or at least making it better with the named args), otherwise optional